django2pydantic/handlers/base.py

```
@@ -244,6 +251,9 @@
     @property
     def is_required(self) -> bool:
         """Return whether the field is required."""
+        # TODO(phuongfi91): null is database only, blank is for validation
```

Ruff (0.8.2)

Line contains TODO, consider resolving the issue

(FIX002)